src/commands/course/_common.ts

```diff
@@ -1,19 +1,48 @@
 import { Cursor, Collection as MongoCollection } from 'mongodb';
 import { Collection } from '../../database';
 import ExecutionContext from '../../ExecutionContext';
+import { bold } from '../../format';
 
 export type Course = {
-  _id: string;
+  _id: number;
```

Comment:
If we're using just the number and not also the course prefix as _id, then it'll create a conflict between courses in different subjects with the same number.